README.md

```diff
@@ -2,37 +2,36 @@
 
 [![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/cpp/cpp%20-%20client%20-%20ci?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=1611&branchName=master)
 
-This repository is for active development of the Azure SDK for C++.
+This repository is for active development of the Azure SDK for C++. For consumers of the SDK we recommend visiting our [public developer docs](https://azure.github.io/azure-sdk-for-cpp) or our versioned [developer docs](https://azure.github.io/azure-sdk-for-cpp).
 
 ## Getting started
 
-To get started with a specific library, see the **README.md** file located in the library's project folder. You can find service libraries in the `/sdk` directory.
+To get started with a library, see the **README.md** file located in the library's project folder. You can find these library folders grouped by service in the `/sdk` directory.
 
-### Prerequisites
+For tutorials, samples, quick starts, and other documentation, go to [Azure for C++ Developers](https://azure.github.io/azure-sdk-for-cpp).
 
-> CMake version 3.15 is required to build these libraries
+## Packages available
+Each service might have a number of libraries available from each of the following categories:
+* [Client - New Releases](#Client-New-Releases)
+* [Client - Previous Versions](#Client-Previous-Versions)
 
-## Latest Release
+### Client: New Releases
 
-> NOTE: All code in this repro is in development state and should not be used in production.
+New wave of packages that we are announcing as **GA** and several that are currently releasing in **preview**. These libraries follow the [Azure SDK Design Guidelines for C++](https://azure.github.io/azure-sdk/cpp_introduction.html) and share a number of core features such as HTTP retries, logging, transport protocols, authentication protocols, etc., so that once you learn how to use these features in one client library, you will know how to use them in other client libraries. You can learn about these shared features at [Azure::Core](sdk/core/azure-core/README.md).

```

Comment:
You are using "preview" word here. Do we need to change it to beta? You know better, just bringing your attention.

Reply:
Good catch, meant to use beta.  My brain doesn't want to let go of preview